cypress.config.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from 'cypress';
+require('dotenv').config();
+
+type Environment = 'localdev' | 'development' | 'staging';
+
+const environment: Environment =
+  (process.env.NEXT_PUBLIC_ENV as Environment) || 'development';
+
+const baseUrlSites: Record<Environment, string> = {
+  localdev: 'http://localhost:3000',
+  development: 'https://housing-register-development.hackney.gov.uk/',
+  staging: 'https://housing-register-staging.hackney.gov.uk/',
```

Comment:
We probably don't need staging unless we plan on running against it too.

Reply:
Cool updated.